components/ai_chat/core/common/mojom/settings_helper.mojom

```diff
@@ -7,10 +7,22 @@ module ai_chat.mojom;
 
 import "brave/components/ai_chat/core/common/mojom/ai_chat.mojom";
 
+enum OperationResult {
+  Success,
+  InvalidUrl,
+};
+
 interface AIChatSettingsHelper {
   GetPremiumStatus() => (PremiumStatus status, PremiumInfo? info);
   GetModelsWithSubtitles() => (array<ModelWithSubtitle> models);
   GetManageUrl() => (string url);
+  GetCustomModels() => (array<Model> models);
+  AddCustomModel(Model model) => (OperationResult result);
+  SaveCustomModel(uint32 index, Model model) => (OperationResult result);
+  DeleteCustomModel(uint32 index);
+  SetDefaultModel(string key);
+  GetDefaultModel() => (string key);
```

Comment:
nit: `GetDefaultModelKey` - everything that has Model in its name in this interface uses the `Model` type

Reply:
changed